plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -512,6 +513,22 @@ protected TestTable createAggregationTestTable(String name, List<String> rows)
         return new TestTable(onRemoteDatabase(), name, "(short_decimal decimal(9, 3), long_decimal decimal(30, 10), t_double double precision, a_bigint bigint)", rows);
     }
 
+    @Test
+    public void testBigintAvgPushdown()
```

Comment:
As far as I can see there are no connectors that support aggregation pushdown (in general) but not `avg(bigint)`?
Hence IMO we should remove the SUPPORTS_AVG_BIGINT flag since SUPPORTS_AGGREGATION_PUSHDOWN covers it. (My reasoning is that count, sum, avg, min/max aggregation pushdown are simple to implement and any connector that supports aggregation pushdown already has impl for those so no point in splitting them further IMO).

Also that means this test isn't adding useful coverage since this seems covered in testNumericAggregationPushdown and testAggregationPushdown in more depth (empty tables, double, bigint, long and short decimal inputs).

Maybe we don't need this additional test?

Reply:
@findepi @wendigo @grantatspothero WDYT?

Reply:
I was hoping this test would cover cases where someone implements `ImplementAvgBigint` incorrectly (for example, not casting the bigint to a double in the pushed aggregate expression resulting in rounded results).

However, there is already a test that does exactly what I described above. I verified if I modify the mysql `ImplementAvgBigint` implementation to the naive implementation `avg(%s)` this test fails:
https://github.com/trinodb/trino/blob/master/testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java#L294-L296 

Thus yup I agree with you, I think removing this test and the associated testing behavior makes sense.